programs/locker/src/instructions/cancel_vesting_escrow.rs

```diff
@@ -0,0 +1,132 @@
+use anchor_spl::token::{Token, TokenAccount};
+use solana_program::program::invoke_signed;
+use spl_token::instruction::close_account;
+
+use crate::*;
+use crate::util::token::transfer_to_recipient;
+
+/// Accounts for [locker::cancel_vesting_escrow].
+#[derive(Accounts)]
+#[event_cpi]
+pub struct CancelVestingEscrow<'info> {
+    /// Escrow.
+    #[account(
+        mut,
+        has_one = creator
```

Comment:
signer can specify any address for claim rent fee, pls use `rent_receiver` here

Reply:
updated